src/systems/ec/implementations/react/components/gallery/src/Gallery.jsx

```diff
@@ -32,6 +35,20 @@ export const Gallery = ({
 
       <Button label={viewAllLabel} variant="ghost" data-ecl-gallery-all />
 
+      <div className="ecl-gallery__footer">
+        <Link
+          href="#"
```

Comment:
the href should be a parameter too.
In fact, the icon is also optional (according to the designers), so I think that it woulbe be clearer if we used a parameter of type Link directly, instead of separated ones.
Something similar to https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/implementations/react/components/expandable/src/Expandable.jsx#L41, but with a Link instead of a Button